bin/compile

```diff
@@ -101,13 +101,17 @@ if ! test -e project/build.properties; then
 	output::error <<-EOF
 		Your scala project must include project/build.properties and define sbt.version
 	EOF
+
+	exit 1
```

Comment:
I think we should have two more Hatchet tests for these cases. The tests will be fast to run, since they'll exit early. (IMO we should have Hatchet tests for all/most happy and failing paths customers can encounter.)